services/postgres/templates/storageclass.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if eq .Values.postgres_storage_class "fast" }}
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: fast
+provisioner: kubernetes.io/gce-pd
+parameters:
+  type: pd-ssd
+{{- end }}
```

Comment:
Do we care about this any more? If we want a good backend, wouldn't we just use Cloud SQL?

Reply:
That's probably the right thing, but our current postgres implementations are all using "fast" and putting the postgres volume on SSD currently.